examples/webgl_video_panorama_equirectangular.html

```diff
@@ -62,14 +75,7 @@
 				// invert the geometry on the x-axis so that all of the faces point inward
 				geometry.scale( - 1, 1, 1 );
 
-				var video = document.createElement( 'video' );
-				video.crossOrigin = 'anonymous';
-				video.width = 640;
-				video.height = 360;
-				video.loop = true;
-				video.muted = true;
-				video.src = 'textures/pano.webm';
-				video.setAttribute( 'webkit-playsinline', 'webkit-playsinline' );
+				var video = document.getElementById( 'video' );
 				video.play();
```

Comment:
You can remove this line by using the `autoplay` attribute in `<video>`.

Reply:
Um, with this change, the video does not play in Chrome and Safari anymore. 

Demo: https://jsfiddle.net/3f86dmxc/ (WebM only)

Reply:
Weird... Thanks for trying!